shared/css/design-system-pegasus.scss

```diff
@@ -16,6 +16,8 @@ $border-radius: 4px;
   font-size: 1.25em;
 }
 
+.h5-lookalike-style { @include h5-lookalike-style}
```

Comment:
Adding this here for now for use in markup — the plan for Phase II of the rebrand updates will include similar heading styles for all h1-h6 tags.